src/Session.php

```diff
@@ -1591,6 +1591,15 @@ public static function validateCSRF($data)
     public static function checkCSRF($data)
     {
 
+        $message = __("The action you have requested is not allowed.");
+        $requestToken = $data['_glpi_csrf_token'];
+        if (
+            isset($_SESSION['glpicsrftokens'][$requestToken])
```

Comment:
According to `Session::validateCSRF()`, `$data['_glpi_csrf_token']` may be missing.
```suggestion
        if (
            ($requestToken = $data['_glpi_csrf_token'] ?? null) !== null
            && isset($_SESSION['glpicsrftokens'][$requestToken])
```